go.mod

```diff
@@ -3,30 +3,78 @@ module github.com/timescale/timescale-prometheus
 go 1.14
 
 require (
+	github.com/Azure/azure-sdk-for-go v46.0.0+incompatible // indirect
+	github.com/Azure/go-autorest/autorest v0.11.5 // indirect
+	github.com/Azure/go-autorest/autorest/adal v0.9.3 // indirect
+	github.com/Azure/go-autorest/autorest/to v0.4.0 // indirect
+	github.com/Azure/go-autorest/autorest/validation v0.3.0 // indirect
 	github.com/NYTimes/gziphandler v1.1.1
 	github.com/OneOfOne/xxhash v1.2.5 // indirect
+	github.com/armon/go-metrics v0.3.4 // indirect
+	github.com/asaskevich/govalidator v0.0.0-20200907205600-7a23bdc65eef // indirect
+	github.com/aws/aws-sdk-go v1.34.19 // indirect
 	github.com/blang/semver/v4 v4.0.0
+	github.com/dgryski/go-sip13 v0.0.0-20190329191031-25c5027a8c7b // indirect
 	github.com/docker/go-connections v0.4.0
 	github.com/edsrzf/mmap-go v1.0.0
 	github.com/go-kit/kit v0.10.0
+	github.com/go-logr/logr v0.2.1 // indirect
+	github.com/go-openapi/runtime v0.19.21 // indirect
+	github.com/go-openapi/spec v0.19.9 // indirect
 	github.com/gogo/protobuf v1.3.1
 	github.com/golang/snappy v0.0.1
-	github.com/grpc-ecosystem/grpc-gateway v1.14.4
+	github.com/google/gofuzz v1.2.0 // indirect
+	github.com/google/pprof v0.0.0-20200905233945-acf8798be1f7 // indirect
+	github.com/googleapis/gnostic v0.5.1 // indirect
+	github.com/gophercloud/gophercloud v0.12.0 // indirect
+	github.com/gregjones/httpcache v0.0.0-20180305231024-9cad4c3443a7 // indirect
+	github.com/grpc-ecosystem/grpc-gateway v1.14.8
+	github.com/hashicorp/consul/api v1.6.0 // indirect
+	github.com/hashicorp/go-hclog v0.14.1 // indirect
+	github.com/hashicorp/go-immutable-radix v1.2.0 // indirect
+	github.com/hashicorp/golang-lru v0.5.4 // indirect
+	github.com/hashicorp/serf v0.9.4 // indirect
+	github.com/imdario/mergo v0.3.5 // indirect
+	github.com/influxdata/influxdb v1.8.2 // indirect
```

Comment:
I don't love this `go.mod change` is there a way we can shrink it?

Reply:
There doesn't seem to be any significant new dependencies...

Reply:
I am not sure that manual intervention will do this. Any suggestions?